scripts/http-form-fuzzer.nse

```diff
@@ -201,15 +234,22 @@ function action(host, port)
       for _,form_plain in ipairs(all_forms) do
         local form = http.parse_form(form_plain)
         if form and form.action then
-          local affected_fields = fuzz_form(form, minlen, maxlen, host, port, path)
+          local affected_fields = fuzz_form(form, minlen, maxlen, timeLimit, end_time, host, port, path)
           if #affected_fields > 0 then
             affected_fields["name"] = "Path: "..path.." Action: "..form["action"]
             table.insert(return_table, affected_fields)
           end
         end
+        if (timeLimit and nmap.clock_ms() > end_time) then
+          break
+        end
       end
     end
+    if (timeLimit and nmap.clock_ms() > end_time) then
+      local timeout_str = "script timed out".." with timelimit = "..((stdnse.get_script_args('http-form-fuzzer.timelimit')) or "30m\nUse http-form-fuzzer.timelimit to change the default timeout.")
```

Comment:
If timeLimit is set, then we can just use it directly instead of calling `get_script_args` again. Use `string.format` instead of concatenation to make it a little clearer. No need to mention how to change the timeout: users should be used to checking the docs or --script-help.